Package created for Arch and tests not passing

I'm writing to say that after the license addition, I was able to package it to Arch Linux and send it to AUR: https://aur.archlinux.org/packages/rdap-client

Besides that, tests don't seem to be passing:

```go test -v
=== RUN   TestASPrint
--- PASS: TestASPrint (0.00s)
=== RUN   TestAsToTextWithErrorOnWriter
--- PASS: TestAsToTextWithErrorOnWriter (0.00s)
=== RUN   TestDomainPrint
--- PASS: TestDomainPrint (0.00s)
=== RUN   TestEntityPrint
--- PASS: TestEntityPrint (0.00s)
=== RUN   TestIPNetPrint
    ipnet_test.go:114:
    ipnet_test.go:114:   inetnum:       200.160.3.0/24
    ipnet_test.go:114:   handle:        200.160.3.0/24
    ipnet_test.go:114:   parent-handle: 200.160.0.0/16
    ipnet_test.go:114:   aut-num:       1234
    ipnet_test.go:114:   start-address: 200.160.3.0
    ipnet_test.go:114:   end-address:   200.160.3.255
    ipnet_test.go:114:   ip-version:    v4
    ipnet_test.go:114:   name:          Crazy Organization
    ipnet_test.go:114:   type:          DIRECT ALLOCATION
    ipnet_test.go:114:   country:       BR
    ipnet_test.go:114:   status:        active
    ipnet_test.go:114: +
    ipnet_test.go:114: +
    ipnet_test.go:114:   inetrev:       200.160.3.0/24
    ipnet_test.go:114:   nserver:       a.dns.br
    ipnet_test.go:114:   nserver:       b.dns.br
    ipnet_test.go:114:   created:       20150301
    ipnet_test.go:114:   changed:       20150310
    ipnet_test.go:114:
    ipnet_test.go:114:   handle:   XXXX
    ipnet_test.go:114:   person:   Joe User
    ipnet_test.go:114:   e-mail:   joe.user@example.com
    ipnet_test.go:114:   address:  Av Naçoes Unidas, 11541, 7 andar, Sao Paulo, SP, 04578-000, BR
    ipnet_test.go:114:   phone:    tel:+55-11-5509-3506;ext=3506
    ipnet_test.go:114:   created:  20150301
    ipnet_test.go:114:   changed:  20150310
    ipnet_test.go:114:
    ipnet_test.go:114:
    ipnet_test.go:116: error
--- FAIL: TestIPNetPrint (0.00s)
FAIL
exit status 1
FAIL    github.com/registrobr/rdap-client/output        0.004s
```

Thanks,

Kemel Zaidan

--- a/output/ipnet_tmpl.go
+++ b/output/ipnet_tmpl.go
@@ -34,8 +34,8 @@ country:       {{.IPNetwork.Country}}
 status:        {{.}}
 {{end}}\
 {{range .IPNetwork.ReverseDelegations}}\
-{{ $startAddress := .StartAddress}}
-{{ $endAddress := .EndAddress }}
+{{- $startAddress := .StartAddress -}}
+{{- $endAddress := .EndAddress -}}
 inetrev:       {{inetnum $startAddress $endAddress}}
 {{range .Nameservers}}\
 nserver:       {{.LDHName}}

